.gitignore

```diff
@@ -29,3 +29,4 @@ env
 .tox/
 .env
 doc/_build
+examples/**/uv.lock
```

Comment:
Lock files should generally be committed to help with reproducible environments :)